app/controllers/recipes_controller.rb

```diff
@@ -3,13 +3,18 @@ def index
     @recipes = Recipe.all
   end
 
+  def show
+    @recipe = Recipe.find(params[:id])
+  end
+
   def new
-    @current_user = User.first
     @recipe = Recipe.new
   end
 
```

Comment:
### Optional
It is better to remove @current_user = User.first, You can access the current logged in user  by using 
current_user helper in new recipe views